proof/invariant-abstract/ARM_HYP/ArchTcbAcc_AI.thy

```diff
@@ -169,6 +169,11 @@ lemma tcb_context_update_aux: "arch_tcb_context_set (P (arch_tcb_context_get atc
 
 
 
+lemma thread_set_valid_arch_state[TcbAcc_AI_assms]:
```

Comment:
Incredibly minor, but while you're here you might as well remove the whitespace above this lemma and make it consistent with the other architectures.

Reply:
well spotted, done